### ISSUE TITLE
SearchWithMinCompatibleSearchNodeIT fails because of deprecation warnings

### ISSUE THREAD

**Issue by astefan** [elastic/elasticsearch#67571]
**Build scan**: https://gradle-enterprise.elastic.co/s/qk6vmzsccmbpw

**Repro line**: `gradlew ':qa:mixed-cluster:v6.8.1#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.SearchWithMinCompatibleSearchNodeIT" -Dtests.method="testMinVersionAsNewVersion" -Dtests.seed=8F15707A436B0076 -Dtests.security.manager=true -Dtests.bwc=true -Dtests.locale=da-DK -Dtests.timezone=Pacific/Pitcairn -Dtests.distribution=default -Druntime.java=8`

**Reproduces locally?**: Yes

**Applicable branches**: 7.x

**Failure excerpt**:

```
org.elasticsearch.backwards.SearchWithMinCompatibleSearchNodeIT > testMinVersionAsNewVersion FAILED
    org.elasticsearch.client.WarningFailureException: method [HEAD], host [http://[::1]:41189], URI [/test_min_version], status line [HTTP/1.1 404 Not Found]
    Warnings: [[types removal] The parameter include_type_name should be explicitly specified in get indices requests to prepare for 7.0. In 7.0 include_type_name will default to 'false', which means responses will omit the type name in mapping definitions.]
        at __randomizedtesting.SeedInfo.seed([8F15707A436B0076:E0820AF121AE21D6]:0)
        at org.elasticsearch.client.RestClient.convertResponse(RestClient.java:322)
        at org.elasticsearch.client.RestClient.performRequest(RestClient.java:296)
        at org.elasticsearch.client.RestClient.performRequest(RestClient.java:270)
        at org.elasticsearch.backwards.SearchWithMinCompatibleSearchNodeIT.prepareTestData(SearchWithMinCompatibleSearchNodeIT.java:69)
```

**Comment by ywelsch** [comment]
@astefan what was the reason for marking this as @elastic/es-distributed?

**Comment by astefan** [comment]
@ywelsch I assumed GET /index related operations fall under the Distributed area. The test itself is Search specific, but the deprecation warnings associated with GET index I think are not. If this is the wrong label, I apologize.

**Comment by ywelsch** [comment]
It's related to `include_type_name`, which relates to search/mapping

**Comment by romseygeek** [comment]
Closed by #67573
